packages/phone-number-privacy/signer/test/integration/domain.test.ts

```diff
@@ -61,12 +61,13 @@ describe('domain', () => {
 
   const signatureRequest = async (
     _domain?: SequentialDelayDomain,
-    _nonce?: number
+    _nonce?: number,
+    keyVersion: number = config.keystore.keys.domains.latest
   ): Promise<[DomainRestrictedSignatureRequest<SequentialDelayDomain>, ThresholdPoprfClient]> => {
     const domain = _domain ?? authenticatedDomain()
     const thresholdPoprfClient = new ThresholdPoprfClient(
-      Buffer.from(TestUtils.Values.DOMAINS_DEV_ODIS_PUBLIC_KEY, 'hex'),
-      Buffer.from(TestUtils.Values.DOMAINS_DEV_ODIS_POLYNOMIAL, 'hex'),
+      Buffer.from(TestUtils.Values.DOMAINS_THRESHOLD_DEV_PUBKEYS[keyVersion - 1], 'hex'), // TODO(2.0.0) should this be in base64?
```

Comment:
also yes looks like this should be 'base64' (only the PUBKEY), but isn't actually used in blinding/unblinding 😬 🙃